.github/workflows/native-docker-validator.yml

```diff
@@ -0,0 +1,36 @@
+name: Native docker validator
+on:
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  validate_native_docker:
+    runs-on: ubuntu-latest
+    if: github.actor != 'dependabot[bot]'
```

Comment:
we could probably also add github actions here to check it isnt the auttomerge glow